_content/monographs/zen-texts_mcrea-tokiwa-yoshida-heine.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Zen Texts"
```

Comment:
For the filename, when there's a large number of authors like this, I'll usually put the publisher (`_bdk.md`)